bootstrap/result.go

```diff
@@ -65,6 +65,9 @@ func (sr *shardResult) AddSeries(id string, rawSeries m3db.DatabaseSeriesBlocks)
 
 // AddResult adds a shard result.
 func (sr *shardResult) AddResult(other m3db.ShardResult) {
+	if other == nil {
+		return
+	}
```

Comment:
Correct, adding a nil result should be a noop. This is called from the base bootstrapper to merge results from different bootstrappers, so if one bootstrapper returns nil, it should be treated as a no-op.